guides/operations/setup/installing_sqream_with_monit.rst

```diff
@@ -103,10 +105,559 @@ Note the following:
+         $ #                    }
+         $ #
+         $ #SQREAM6-START
+         $ #check process sqream6 with pidfile /var/run/sqream6.pid
+         $ #start program = "/usr/bin/systemctl start sqream6"
+         $ #stop program = "/usr/bin/systemctl stop sqream6"
+         $ #SQREAM6-END
+         $ #       alert user@domain.com on {nonexist, timeout}
+         $ #                      with mail-format {
+         $ #                            from:     Monit@$HOST
+         $ #                            subject:  sqream2 $EVENT - $ACTION
+         $ #                            message:  This is an automate mail, sent from monit.
+         $ #                    }
+         $ #
+         $ #SQREAM7-START
+         $ #check process sqream7 with pidfile /var/run/sqream7.pid
+         $ #start program = "/usr/bin/systemctl start sqream7"
+         $ #stop program = "/usr/bin/systemctl stop sqream7"
+         $ #SQREAM7-END
+         $ #                      with mail-format {
+         $ #                            from:     Monit@$HOST
+         $ #                            subject:  sqream2 $EVENT - $ACTION
+         $ #                            message:  This is an automate mail, sent from monit.
+         $ #                    }
+         $ #
+         $ #SQREAM8-START
+         $ #check process sqream8 with pidfile /var/run/sqream8.pid
+         $ #start program = "/usr/bin/systemctl start sqream8"
+         $ #stop program = "/usr/bin/systemctl stop sqream8"
+         $ #SQREAM8-END
+         $ #       alert user@domain.com on {nonexist, timeout}
+         $ #                      with mail-format {
+         $ #                            from:     Monit@$HOST
+         $ #                            subject:  sqream2 $EVENT - $ACTION
+         $ #                            message:  This is an automate mail, sent from monit.
+         $ #                    }
+
+Starting Monit
+====================================  
+
+**To start Monit:**
+
+1. Stop all actively running SQream services:
+
+   .. code-block:: console
+     
+      $ sudo systemctl stop sqream[1-4]  #This command stops sqream1, sqream2,..,sqream4
+      $ sudo systemctl stop serverpicker
+      $ sudo systemctl stop metadataserver
+
+2. Start Monit as a super user:
+
+   .. code-block:: console
+     
+      $ sudo systemctl start monit
+   
+3. Verify that the following SQream processes are running and listening:
+  
+   .. code-block:: console
+     
+      $ sudo systemctl status metadataserver
+      $ sudo systemctl status serverpicker
+      $ sudo systemctl status sqream1  #... etc
+      $ sudo netstat -nltp   #to see that sqream is listening on ports  
+ 
+4. View Monit's service status:
+
+   .. code-block:: console
+     
+      $ sudo systemctl status monit
+
+5. If all good, enable the Monit service to start on boot: **Comment - What status result(s) are we looking for?**       
```

Comment:
I don't know what is expected but a good idea to show an example

Reply:
Do I need access to the SQream GUI to test this? @Sean-Tomarian

Reply:
No, this is a Linux command (not SQream)